`make container` is broken after the recent base image bump

This happened after https://github.com/GoogleCloudPlatform/netd/pull/124 is merged.

Try building locally and hit the below error:
```
...
 => CACHED [2/5] RUN apt-get -y update                                                                                                                  0.0s
 => ERROR [3/5] RUN apt-get -y install curl jq bash iproute2                                                                                            1.0s
------                                                                                                                                                       
 > [3/5] RUN apt-get -y install curl jq bash iproute2:                                                                                                       
#7 0.261 Reading package lists...                                                                                                                            
#7 0.713 Building dependency tree...                                                                                                                         
#7 0.820 Reading state information...                                                                                                                        
#7 0.871 Some packages could not be installed. This may mean that you have                                                                                   
#7 0.871 requested an impossible situation or if you are using the unstable
#7 0.871 distribution that some required packages have not yet been created
#7 0.871 or been moved out of Incoming.
#7 0.871 The following information may help to resolve the situation:
#7 0.871 
#7 0.871 The following packages have unmet dependencies:
#7 0.926  iproute2 : Depends: libcap2 (>= 1:2.10) but it is not going to be installed
#7 0.926  libcap2-bin : Depends: libcap2 (>= 1:2.33) but it is not going to be installed
#7 0.926                Recommends: libpam-cap but it is not installable
#7 0.938 E: Unable to correct problems, you have held broken packages.
------
.dockerfile-amd64:20
--------------------
  18 |     
  19 |     RUN apt-get -y update
  20 | >>> RUN apt-get -y install curl jq bash iproute2
  21 |     ADD scripts/install-cni.sh /install-cni.sh
  22 |     ADD bin/amd64/netd /netd
--------------------
error: failed to solve: rpc error: code = Unknown desc = process "/bin/sh -c apt-get -y install curl jq bash iproute2" did not complete successfully: exit code: 100
make: *** [Makefile:157: .container-gcr.io_gke-release-staging_netd-amd64-v0.4.1-dirty] Error 1
```

cc @jingyuanliang @sypakine  @wespanther

--- a/Dockerfile.in
+++ b/Dockerfile.in
@@ -17,6 +17,11 @@ FROM ARG_FROM
 MAINTAINER Zihong Zheng <zihongz@google.com>
 
 RUN apt-get -y update
+
+# Necessary to install iproute2
+# https://github.com/GoogleCloudPlatform/netd/issues/125
+RUN apt-mark unhold libcap2
+
 RUN apt-get -y install curl jq bash iproute2
 ADD scripts/install-cni.sh /install-cni.sh
 ADD bin/ARG_ARCH/ARG_BIN /ARG_BIN

